packages/send-emails/backend/node/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "nylas-send-emails-node",
+  "name": "nylas-send-emails-node-express",
```

Comment:
Nit: I don't think we need to include express in the name of the example, since the frontend dropbox will only indicate Node.

Reply:
lol didn't even notice that will change